frontend/appflowy_flutter/lib/plugins/database_view/application/field/type_option/date_bloc.dart

```diff
@@ -54,9 +54,9 @@ class DateTypeOptionBloc
         typeOption.timeFormat = timeFormat;
       }
 
-      if (includeTime != null) {
-        typeOption.includeTime = includeTime;
-      }
+      // if (includeTime != null) {
+      //   typeOption.includeTime = includeTime;
+      // }
```

Comment:
```suggestion
```

remove commented code if it isn't being used.

Reply:
Deleted, resolving thread